lib/CryptoLD.js

```diff
@@ -0,0 +1,115 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+/**
+ * Maps key type id to the current/recommended key suite for that type.
```

Comment:
It could very well be an oversight, but where in the documentation do we state that we have recommended key suites for each key type?

Reply:
@gannan08 it's mentioned in the Readme, but I'm open to suggestions as to how to make it more explicit.